src/app/error.tsx

```
@@ -0,0 +1,31 @@
+"use client";
+
+import { useRouter } from "next/navigation";
+import { FixedLayout } from "./_components";
+import { ErrorIcon } from "../icons";
+
+export default function Error() {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)